### PR TITLE
adding cards for 2016 and 2017/18 stop production

### PR DESCRIPTION
**Pull request by gabhijith**
Here We are adding the 2016 and 2017/18 cards for stop production.

in bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV we have the 2017/18 cards
and in bin/MadGraph5_aMCatNLO/cards/production/13TeV we have the 2016 Cards.

We added 1 example mass point and templates to create more.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/st_jj_rpv_LO_M1-1000/st_jj_rpv_LO_M1-1000_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+ 0   = xetamin ! minimum rapidity for two jets in the WBF case  
+ 0   = deltaeta ! minimum rapidity for two jets in the WBF case 
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 5 = maxjetflavor    ! Maximum jet pdg code
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/st_jj_rpv_LO_M1-1000/st_jj_rpv_LO_M1-1000_run_card.dat:237]: lhapdf:292000 is a 4f pdf NNPDF30_nlo_nf_4_pdfas.
But the settings are inconsistent

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/st_jj_rpv_LO_M1-1000/st_jj_rpv_LO_M1-1000_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 1        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
+ 1        = highestmult      ! for ickkw=2, highest mult group
+ 1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+ 1        = alpsfact         ! scale factor for QCD emission vx
+ F        = chcluster        ! cluster only according to channel diag
+ F        = pdfwgt           ! for ickkw=1, perform pdf reweighting
+ 5        = asrwgtflavor     ! highest quark flavor for a_s reweight
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/st_jj_rpv_LO_M1-1000/st_jj_rpv_LO_M1-1000_run_card.dat:73]: lhapdf:292000 is a 4f pdf NNPDF30_nlo_nf_4_pdfas.
So this setting is inconsistent :   5 = maxjetflavor    ! Maximum jet pdg code

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/templates/PROCESSNAME_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~ b b~
+define j = g u c d s u~ c~ d~ s~ b b~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/templates/PROCESSNAME_proc_card.dat:11]: This is default, no? Please remove if so. You define q below but don't use it, or?

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/templates/PROCESSNAME_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                  
+  292000 = lhaid
```
Comment (agrohsje) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/templates/PROCESSNAME_run_card.dat:53]: As above.

**Comment by agrohsje** [comment]
Suggest to just keep the templates and remove the rest. Main comment is about PDF and redundant info. Please fix. Is this aiming at legacy or pre-legacy production?

**Comment by sihyunjeon** [comment]
This is for prelegacy.

The requestors agreed to use NNPDF31 + CP5 for the whole years.
I'll ask them to clean up things.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/templates/PROCESSNAME_proc_card.dat (hunk 3 above)
Comment (gabhijith) [reply bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_jj_rpv_LO/templates/PROCESSNAME_proc_card.dat:11]: Yes, I have removed the 
```
set group_subprocesses Auto
set ignore_six_quark_processes False
set loop_optimized_output True
set complex_mass_scheme False
```
And also removed the quark part.

**Comment by gabhijith** [comment]
> This is for prelegacy.
> 
> The requestors agreed to use NNPDF31 + CP5 for the whole years.
> I'll ask them to clean up things.

I have cleaned up cards with PDF of 292000. As @sihyunjeon mentioned we are using NNPDF31 + CP5 for the whole years.

**Comment by agrohsje** [comment]
Why do you want to push this to 26x branch? This isn't updated since long. Why not using master with 2.6.5 and all the fixes?

**Comment by sihyunjeon** [comment]
@agrohsje 
Indeed, I had to apply the thread patches to gridpacks, and there was no issues with gen checking scripts.
Also I told the analysers to use the master branch for gridpack generations, would this be ok?
These are the RPVstopstop samples that I tried to inject in this week's MCCM.

**Pull request merged**